src/include/sof/schedule/task.h

```diff
@@ -41,18 +42,26 @@ enum task_state {
 	SOF_TASK_STATE_RESCHEDULE,
```

Comment:
Do we still need `SOF_TASK_STATE_RESCHEDULE`, I see you removed it from EDF. Some functions still return that value, but do LL do anything with that?

Reply:
Yes, it's used by LL scheduler and also will be used by EDF in the near future.